### ISSUE TITLE
Incorrect statement on failure detector

### ISSUE THREAD

**Issue by johanandren** [akka/akka#27751]
The docs on the failure detector says:
```
Remote DeathWatch uses heartbeat messages and the failure detector to 

* detect network failures and JVM crashes
* generate the `Terminated` message to the watching actor on failure 
* gracefully terminate of watched actors
```

But it does in fact not at all "gracefully terminate watched actors", we should remove or restate in a correct way (whatever it was intended to mean)

**Comment by patriknw** [comment]
Yup, that is wrong. Also the second is wrong within a cluster.

**Comment by patriknw** [comment]
I’m pretty sure I fixed this in one of the 2.6 cleanups
